app/mailers/provider_user_mailer.rb

```diff
@@ -82,7 +82,7 @@ def domain(account)
   # private
 
   # def admin_url(user)
-  #   admin_dashboard_url(:host => user.account.provider_account.domain)
+  #   admin_dashboard_url(:host => user.account.provider_account.external_admin_domain)
   # end
```

Comment:
If this is not used, perhaps we could directly remove it 😄

Reply:
I prefer letting it there, not the purpose of this PR
Maybe somebody will take a look at it